user/create.js

```diff
@@ -0,0 +1,26 @@
+import * as dynamoDbLib from '../libs/dynamodb-lib';
+
+export const main = async event => {
+  /**
+   * @property {string} userId
+   * @property {string} email
+   */
+	const { sub, email } = event.request.userAttributes;
+
+	const params = {
+		TableName: process.env.USERS_TABLE_NAME,
+		Item: {
+			userId: sub,
+			email: email
+		}
+	};
+
+	try {
+		await dynamoDbLib.call('put', params);
+		console.log(params.Item);
```

Comment:
If the console.log is not necessary, I would remove it. 

I would also suggest checking the indentation. It looks like 4 spaces, but we are using 2 everywhere. If you are using Visual Studio Code, I would turn on `FormatOnSave` option. Or run the `Reformat` action.

Reply:
I have VS Code(with FormatOnSave). Indentation looks good in my editor.

Reply:
Ok. Maybe GitHub is displaying things differently